lib/auth/apiserver.go

```diff
@@ -123,9 +123,9 @@ func NewAPIServer(config *APIConfig) (http.Handler, error) {
 	// Servers and presence heartbeat
 	srv.POST("/:version/namespaces/:namespace/nodes/keepalive", srv.WithAuth(srv.keepAliveNode))
 	srv.POST("/:version/authservers", srv.WithAuth(srv.upsertAuthServer))
-	srv.GET("/:version/authservers", srv.WithAuth(srv.getAuthServers))
+	srv.GET("/:version/authservers", srv.WithScopedAuth(srv.getAuthServers))
 	srv.POST("/:version/proxies", srv.WithAuth(srv.upsertProxy))
-	srv.GET("/:version/proxies", srv.WithAuth(srv.getProxies))
+	srv.GET("/:version/proxies", srv.WithScopedAuth(srv.getProxies))
```

Comment:
What is going on here? There is no mention to scoped authz checks on proxies and auth in the PR description. 

Is this just to prove out that scoped authorization functions on the Auth Rest API as well? Should we move this to a separate change? Is this required for scoped token authz?